lab-davidw/test/server-test.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const server = require('../server');
+const chai = require('chai');
+const http = require('chai-http');
+const expect = chai.expect;
+
+chai.use(http);
+
+describe('server module', function() {
+  before(done => {
+    server.listen(3030);
+    done();
+  });
+  after(done => {
+    server.close();
+    done();
+  });
+});
+
+describe('GET method', function() {
+  let resource;
+  before(done => {
+    chai.request(server)
+    .post('/api/lure')
+    .send({name: 'momba', type: 'rattler', targets: 'trout'})
+    .end((err, res) => {
+      resource = JSON.parse(res.text.toString());
+      done();
+    });
+  });
+  after(done => {
+    chai.request(server)
+    .delete('/api/lure')
+    .query({id: resource.id})
+    .end((err, res) => {
+      console.log(res);
+      done();
+    });
+  });
+
```

Comment:
the server setup before/after looks ok, but you may have to move the after block all the way down. also, if you're doing TDD, you'd probably want to POST something before trying to GET it, else nothing will be available.